include/class.ostsession.php

```diff
@@ -200,7 +202,9 @@ function update($id, $data){
         $ttl = $this && method_exists($this, 'getTTL')
             ? $this->getTTL() : SESSION_TTL;
 
-        assert($this->data->session_id == $id);
+        // Create a session data obj if not loaded.
+        if (!$this->data)
+            $this->data = new SessionData(['session_id' => $id]);
```

Comment:
Actually, never mind. The ORM does not support updating the primary key at the moment